automod/rules/replies.go

```diff
@@ -27,3 +27,24 @@ func ReplyCountPostRule(evt *automod.RecordEvent, post *appbsky.FeedPost) error
 	evt.IncrementDistinct("reply-to", did, parentURI.Authority().String())
 	return nil
 }
+
+var identicalReplyLimit = 5
+
+// Looks for accounts posting the exact same text multiple times. Does not currently count the number of distinct accounts replied to, just counts replies at all.
+//
+// There can be legitimate situations that trigger this rule, so in most situations should be a "report" not "label" action.
+func IdenticalReplyPostRule(evt *automod.RecordEvent, post *appbsky.FeedPost) error {
+	if post.Reply == nil || IsSelfThread(evt, post) {
+		return nil
+	}
+
+	// use a specific period (IncrementPeriod()) to reduce the number of counters (one per unique post text)
+	period := automod.PeriodDay
+	bucket := evt.Account.Identity.DID.String() + "/" + HashOfString(post.Text)
+	if evt.GetCount("reply-text", bucket, period) >= identicalReplyLimit {
+		evt.AddAccountFlag("multi-identical-reply")
```

Comment:
Not new to this PR, but I think some docs on the semantic distinctions between `AccountLabels`, `AccountFlags`, and `AccountReports` are needed somewhere (and then perhaps most methods touching them should have a quick pointer to that doc).  The last one of the group is close to self-explanatory, but the first two are both just `[]string` in the code and don't autounpack their meaning very readily.

Reply:
mmm! added doc comments to all the fields for most of the RepoEvent variants